static/gsApp/components/quotaExceededNavItem.spec.tsx

```diff
@@ -0,0 +1,75 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {UserFixture} from 'sentry-fixture/user';
+
+import {SubscriptionFixture} from 'getsentry-test/fixtures/subscription';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import ConfigStore from 'sentry/stores/configStore';
+
+import PrimaryNavigationQuotaExceeded from 'getsentry/components/quotaExceededNavItem';
+import SubscriptionStore from 'getsentry/stores/subscriptionStore';
+
+describe('PrimaryNavigationQuotaExceeded', function () {
+  const organization = OrganizationFixture();
+  const subscription = SubscriptionFixture({
+    organization,
+    plan: 'am3_business',
+  });
+  subscription.categories.errors!.usageExceeded = true;
+  subscription.categories.replays!.usageExceeded = true;
+  subscription.categories.spans!.usageExceeded = true;
+  SubscriptionStore.set(organization.slug, subscription);
```

Comment:
nit: it's best practice to do this setup within a `beforeEach()` to ensure that the state is correct before each test run